### PR TITLE
change ui and add loader

### LINKED ISSUE #4: Improve App UI
- Change UI elements to better represent data
- Add loader to page to show waiting

### PR DESCRIPTION
**Pull request by starboi02**
Fixes #4 
ss1

### CODE REVIEW

File: app/src/main/res/drawable/round_corner.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="#D61A3C" />
```
Comment (Parthav46) [inline app/src/main/res/drawable/round_corner.xml]: move colors to colors.xml and refer it here

File: app/src/main/res/drawable/round_edit.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="#232323" />
```
Comment (Parthav46) [inline app/src/main/res/drawable/round_edit.xml]: same as above

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,30 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
+    android:background="#141414"
```
Comment (Parthav46) [inline app/src/main/res/layout/activity_main.xml]: same as above

File: app/src/main/res/drawable/round_corner.xml (hunk 1 above)
Comment (Parthav46) [reply app/src/main/res/drawable/round_corner.xml]: also use color hex #00B2F9 for the button as it is the primary color for paytm

**Review by Parthav46 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (starboi02) [reply app/src/main/res/layout/activity_main.xml]: @Parthav46 ok sure would do that

File: app/src/main/res/drawable/round_edit.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
 
-    <solid android:color="#232323" />
+    <solid android:color="@color/viewColor" />
```
Comment (Parthav46) [inline app/src/main/res/drawable/round_edit.xml]: colorAccent is the same as viewColor

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -3,4 +3,7 @@
     <color name="colorPrimary">#1b1b1b</color>
     <color name="colorPrimaryDark">#1e1e1e</color>
     <color name="colorAccent">#232323</color>
+    <color name="backgroundColor">#141414</color>
+    <color name="viewColor">#232323</color>
```
Comment (Parthav46) [inline app/src/main/res/values/colors.xml]: remove this

**Review by Parthav46 — APPROVED** [review]

**Pull request merged**